services/m3aggregator/config/aggregator.go

```diff
@@ -46,6 +46,10 @@ import (
 	"github.com/spaolacci/murmur3"
 )
 
+const (
+	initialBufferSizeGrowthFactor = 2
```

Comment:
Do we want to make this twice the flush size? Seems like that might be overkill. As a possible alternative we could make this a function call so we could use a tighter bound. For example:

```
func (c *AggregatorConfiguration) getBufferSize(n int) int {
    // Buffer size should be 1.5 times the maximum flush size to account for overflow.
    return c.MaxFlushSize * 3 * 2
}

```